fmriprep/workflows/base.py

```diff
@@ -129,18 +157,14 @@ def init_fmriprep_wf(layout, subject_list, task_id, echo_idx, run_uuid, work_dir
             run-to-run replicability when used with --omp-nthreads 1
         freesurfer : bool
             Enable FreeSurfer surface reconstruction (may increase runtime)
-        output_spaces : list
-            List of output spaces functional images are to be resampled to.
-            Some parts of pipeline will only be instantiated for some output spaces.
-
-            Valid spaces:
-
-             - T1w
-             - template
-             - fsnative
-             - fsaverage (or other pre-existing FreeSurfer templates)
-        template : str
-            Name of template targeted by ``template`` output space
+        output_spaces : OrderedDict
```

Comment:
Why is the ordering important?

Reply:
Because the first volumetric standard space is taken as "primary". I agree we will get rid of both aspects of an ordered dict (ordering and unique keys). One may want to generate all possible resamplings for a template (for example).
I would leave that for future iterations, though.